google/ggrc-core
# GGRC-8014 Create layout for maintenance mode

# Issue description

What is the expected result / outcome? (ie What should happen?)
GGRC should be stable and refresh itself whenever needed (automatically)
What is the impact if it is not fixed / implemented? (Quantify the impact, eg, # of hours)
Poor user experience and frustration
Seems that user was trying to make actions during website maintenance. We should add a screen that notifies users about site maintenance

# Steps to test the changes

1. Open _maintenance_mode/maintenance.html_ file in the browser.
2. Verify layout. 

# Solution description

1. Created layout for maintenance mode. The page will be set up by BE and returned in case of maintenance.

# Sanity checklist




# PR Review checklist
